modules/apps/account/account-service/src/main/java/com/liferay/account/internal/search/spi/model/query/contributor/AccountEntryModelPreFilterContributor.java

```diff
@@ -123,4 +125,19 @@ private void _filterByStatus(
 		}
 	}
 
+	private void _filterByType(
+		BooleanFilter booleanFilter, SearchContext searchContext) {
+
+		String[] types = GetterUtil.getStringValues(
+			searchContext.getAttribute(Field.TYPE));
```

Comment:
We don't really need to have an array here since there's only 2 types. We can keep this if we'll have more than 2 types in the future and want to provide a filter for multiple accounts.